snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/StepExecutionSupport.java

```diff
@@ -128,6 +128,7 @@ protected void failExecution() {
      */
     @Override
     public ExitStatus afterStep(StepExecution stepExecution) {
+        this.stepExecution = stepExecution;
```

Comment:
Was this a bug? It doesn't seem related to the Java 11 transition.

Reply:
It was causing a problem with one of the tests.  Probably related to the easymock upgrade.